### PR TITLE
[hwcomposer] Add envvars to set buffercount and force gles comp. Cont…

### PR DESCRIPTION
**Pull request by sletta**
…ributes to JB#31910.

### CODE REVIEW

File: hwcomposer/hwcomposer_backend_v11.cpp [hunk 1]
```diff
@@ -72,6 +71,8 @@ HWComposer::HWComposer(unsigned int width, unsigned int height, unsigned int for
     , mlist(mList)
     , num_displays(num_displays)
 {
+    int bufferCount = qBound(2, qgetenv("QPA_HWC_BUFFER_COUNT").toInt(), 8);
```
Comment (rburchell) [inline hwcomposer/hwcomposer_backend_v11.cpp]: Does qBound'ing the value make sense - as opposed to just allowing any value for testing purposes? (I actually don't know the answer :))

**Comment by rburchell** [comment]
LGTM

**Comment by mlehtima** [comment]
LGTM, tested on iyokan (Xperia Pro).

File: hwcomposer/hwcomposer_backend_v11.cpp (hunk 1 above)
Comment (sletta) [inline hwcomposer/hwcomposer_backend_v11.cpp]: Just a hint of protection against bizarre input, I figured.. Nobody would ever want to use more than 4-5 buffers anyway, so 8 is already a very high number :p

**Pull request merged**